include/boost/json/system_error.hpp

```diff
@@ -42,16 +36,6 @@ generic_category();
 using boost::system::generic_category;
 #endif
 
-#else
-
-using error_code = std::error_code;
-using error_category = std::error_category;
-using error_condition = std::error_condition;
-using system_error = std::system_error;
-using std::generic_category;
-
-#endif
```

Comment:
Boost.JSON requires C++11.
Why do you prefer the Boost instead of the features of the C++11 standard library?

Reply:
As you can see, I simply removed the preprocessor logic for standalone mode. I presume, Boost.Align was used because some of the supported platforms do not have implementation for std::align and std::max_align_t. That is, some of the supported platforms implement only a subset of C++11.